packages/teams/src/utils/update-template-based-on-privs.ts

```diff
@@ -0,0 +1,23 @@
+import { IGroupTemplate } from "../types";
+import { IUser } from "@esri/arcgis-rest-types";
+import { cloneObject, includes } from "@esri/hub-common";
+
+export function updateTemplateBasedOnPrivs(
```

Comment:
comment needed. also can we name this something that connects it to the `privPropValues` hash? i.e. `applyPrivPropValues(..)`?